video/out/opengl/video.c

```diff
@@ -3391,7 +3393,8 @@ static void check_gl_features(struct gl_video *p)
     bool have_mglsl = gl->glsl_version >= 130; // modern GLSL (1st class arrays etc.)
     bool have_texrg = gl->mpgl_caps & MPGL_CAP_TEX_RG;
     bool have_tex16 = !gl->es || (gl->mpgl_caps & MPGL_CAP_EXT16);
-    bool have_compute = gl->glsl_version >= 430; // easiest way to ensure all
+    bool have_compute = gl->DispatchCompute;
```

Comment:
I don't think it would be a big deal, only a one line definition that updates `gl->mpgl_caps`. Seems to fit right next to this one:

```glsl
    if (is_software_gl(gl)) {
        gl->mpgl_caps |= MPGL_CAP_SW;
        mp_verbose(log, "Detected suspected software renderer.\n");
    }
```

I guess it depends on how often we're going to be accessing these things. I would certainly love to try and add more compute-shader optimized drop-in-replacement routines in the future, so any additional check that we can reduce to a single flag comparison would benefit from it. And since we're already checking for three extensions in total (compute_shader, image_load_store and array_of_arrays) that's worth it, I think.